<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>64</width>
    <height>64</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="totalFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1201</width>
      <height>600</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="headFrame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-1</y>
       <width>1201</width>
       <height>75</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(41, 42, 55);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="lblTitle">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>5</y>
        <width>341</width>
        <height>64</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕코딩</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>EMS Iot Dashboard</string>
      </property>
     </widget>
     <widget class="QFrame" name="iconFrame">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>5</y>
        <width>64</width>
        <height>64</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/iot);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="greetingFrame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>75</y>
       <width>300</width>
       <height>500</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(60, 60, 90);
border-style : outset;
border-width : 5px;
border-color : rgb(50, 50, 180);
border-radius : 10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="weatherFrame">
      <property name="geometry">
       <rect>
        <x>118</x>
        <y>180</y>
        <width>64</width>
        <height>64</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/sun);
border : none</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lblDate">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕코딩</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 170, 0);
border : none;</string>
      </property>
      <property name="text">
       <string>9999-99-99</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblDay">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕코딩</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 170, 0);
border : none;</string>
      </property>
      <property name="text">
       <string>Wednesday</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblTime">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕코딩</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 170, 0);
border : none;</string>
      </property>
      <property name="text">
       <string>99-99</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblGreeting">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>300</y>
        <width>270</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕코딩</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(208, 208, 208);
border : none;</string>
      </property>
      <property name="text">
       <string>Good Morning!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="tempFrame">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>75</y>
       <width>300</width>
       <height>500</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(63, 19, 5);
border-style : outset;
border-width : 5px;
border-color : rgb(180, 50, 50);
border-radius : 10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="weatherFrame_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>128</width>
        <height>128</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/temp32);
background-repeat: no-repeat;
border : none</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lblCurrTemp">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>230</y>
        <width>161</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕코딩</family>
        <pointsize>48</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 100, 25);
border : none;</string>
      </property>
      <property name="text">
       <string>99.9</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblTempTitle">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕코딩</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(191, 191, 191);
border : none;</string>
      </property>
      <property name="text">
       <string>EMS05 Temperature</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QDial" name="dial">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>350</y>
        <width>128</width>
        <height>128</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 156, 34);</string>
      </property>
      <property name="notchesVisible">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblCurrTemp_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>230</y>
        <width>101</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕코딩</family>
        <pointsize>48</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 100, 25);
border : none;</string>
      </property>
      <property name="text">
       <string>℃</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <zorder>weatherFrame_3</zorder>
     <zorder>lblTempTitle</zorder>
     <zorder>dial</zorder>
     <zorder>lblCurrTemp_2</zorder>
     <zorder>lblCurrTemp</zorder>
    </widget>
    <widget class="QFrame" name="tempFrame_2">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>75</y>
       <width>300</width>
       <height>500</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 50, 115);
border-style : outset;
border-width : 5px;
border-color : rgb(0, 50, 180);
border-radius : 10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="weatherFrame_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>128</width>
        <height>128</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/humid);
background-repeat: no-repeat;
border : none</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lblCurrTemp_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>230</y>
        <width>111</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕코딩</family>
        <pointsize>48</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(56, 176, 247);
border : none;</string>
      </property>
      <property name="text">
       <string>99</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblTempTitle_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕코딩</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(191, 191, 191);
border : none;</string>
      </property>
      <property name="text">
       <string>EMS05 Humidity</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblCurrTemp_4">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>190</y>
        <width>101</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>휴먼옛체</family>
        <pointsize>48</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(56, 176, 247);
border : none;</string>
      </property>
      <property name="text">
       <string>%</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QDial" name="dial_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>350</y>
        <width>128</width>
        <height>128</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(44, 217, 255);</string>
      </property>
      <property name="notchesVisible">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblCurrTemp_5">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>230</y>
        <width>101</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>새굴림</family>
        <pointsize>48</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(56, 176, 247);
border : none;</string>
      </property>
      <property name="text">
       <string>%</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <zorder>weatherFrame_4</zorder>
     <zorder>lblTempTitle_2</zorder>
     <zorder>lblCurrTemp_4</zorder>
     <zorder>dial_2</zorder>
     <zorder>lblCurrTemp_5</zorder>
     <zorder>lblCurrTemp_3</zorder>
    </widget>
    <zorder>tempFrame_2</zorder>
    <zorder>headFrame</zorder>
    <zorder>greetingFrame</zorder>
    <zorder>tempFrame</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../images/IoT.qrc"/>
 </resources>
 <connections/>
</ui>
